vassal-app/src/main/java/VASSAL/tools/imageop/AbstractOpImpl.java

```diff
@@ -32,7 +33,7 @@
 /**
  * An abstract representation of an operation which may be applied to an
  * {@link Image}. <code>ImageOp</code> is the base class for all such
- * operations. The results of all operations are memoized (using a
+ * operations. The results of all operations are memorized (using a
```

Comment:
`memoized` is correct.

https://en.wikipedia.org/wiki/Memoization

Reply:
Never heard of this before, we simpleton practical developers/programmers just call this "caching". My IntelliJ complained about this too. It's a good sign though, IntelliJ only starts complaining about spelling once all errors and warnings from are gone from a class.